Module.java

```diff
@@ -0,0 +1,36 @@
+package extraction.sProject;
+
+public class Module {
+	
+	private String moduleID;
+	private double moduelsScore;
```

Comment:
You put "moduelsScore" instead of "moduleScore" or something like this. Your getters and setters use this value too.